### PR TITLE
The Three Musketeers

### PR DESCRIPTION
**Pull request by Slorbin**
## About The Pull Request

Count them. Three! With additional special text if all three can't roll at once.

They get pretty good musket skill. They only spawn as a migrant wave, and only one can spawn! They get far less other skills overall than other combat classes, then.

oh also swashbucklers get their rapier back bc they were balanced specifically around having the rapier.

## Why It's Good For The Game

more guns on funny guys

### CODE REVIEW

**Comment by DeerJesus** [comment]
also, fun little fact, an arquebus and a musket are the same thing. synonyms

File: code/datums/migrants/migrant_waves/musketeer roles.dm [hunk 1]
```diff
@@ -0,0 +1,44 @@
+/datum/migrant_role/musketeer/musketeer
+	name = "Musketeer"
+	greet_text = "A jolly adventurer armed with a musket and a rapier, seeking to save the girl and the day! Don't lose sight of your two companions, you're inseperable!"
```
Comment (Gyrandola) [inline code/datums/migrants/migrant_waves/musketeer roles.dm:3]: Please something more fitting than "jolly adventurer"

Comment (DeerJesus) [reply code/datums/migrants/migrant_waves/musketeer roles.dm:3]: Like what? They're jolly adventurers

**Comment by Frienderanian** [comment]
I would just make it so that you need three to spawn. I get what you're going for- but it isn't hard to get three people. Heartfelts spawn with three.

**Review by Gyrandola — APPROVED** [review]

**Pull request merged**